lib/remove-project.ts

```diff
@@ -0,0 +1,50 @@
+import config from "./config";
+import consts from "./consts";
+import output from "./output";
+import {
+  getSelectedProjects,
+  getIsUsingComponents,
+} from "./utils/getSelectedProjects";
+import promptForProject from "./utils/promptForProject";
+
+async function removeProject() {
+  const projects = getSelectedProjects();
+  const isUsingComponents = getIsUsingComponents();
+  if (!projects.length && !isUsingComponents) {
+    console.log(
+      "\n" +
+        "No projects found in your workspace.\n" +
+        `Try adding one with: ${output.info("ditto-cli project add")}\n`
+    );
+    return;
+  }
+
+  const allProjects = isUsingComponents
+    ? [{ id: "components", name: "Ditto Component Library" }, ...projects]
+    : projects;
+
+  const projectToRemove = await promptForProject({
+    projects: allProjects,
+    message: isUsingComponents
+      ? "Select a project or library to remove"
```

Comment:
I wonder if we should update our nomenclature in the CLI to refer to "sources" instead of "projects", since places like this are a little bit awkward. "Select a project or library to remove" is a little confusing bc it implies that a "library" is a thing, where really we're only talking about THE component library.

If we were to switch over to saying internally "a project or the component library are a type of 'source' the CLI uses", then this message could say "Select a source to remove" and be less ambiguous.

Something for us to noodle on 🤔